config/routes.rb

```diff
@@ -26,6 +26,8 @@
     post "/subscribers/auth-token", to: "subscribers_auth_token#auth_token"
     post "/subscriptions/auth-token", to: "subscriptions_auth_token#auth_token"
 
+    post "/subscribers/from-account-token", to: "subscribers_from_account_token#from_token"
```

Comment:
Minor but I wonder if the from is superfluous here and this can be named more consistently as per auth_token?